en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
+gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This setting does not override the "Distance to WP" option.
+gui.xaero_box_arrow_scale=Scale of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_lock_player_heads=Display player heads instead of colored dots even if the Tab key is not pressed.
+gui.xaero_box_cave_zoom=How much to zoom further in the minimap when the cave mode is active.
+gui.xaero_box_entity_depth=Display entities darker on the minimap depending on their Y level relative to you.
+gui.xaero_box_height_limit=Maximum Y-level distance for entities until they are no longer displayed on the minimap.
+gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.
+gui.xaero_toggle_map_waypoints=Toggle On-Map Waypoints
+gui.xaero_entity_nametags=Entities With Name Tags
 gui.xaero_minimap_text_align=Info Text Alignment
 gui.xaero_center=Center
 gui.xaero_left=Left
 gui.xaero_right=Right
-gui.xaero_show_angles=Show View Angles
-gui.xaero_box_entity_nametags=Display entity name under the dot for entities that have a name tag.
-gui.xaero_box_teleport_default_command=The command used for waypoints teleportation if a server-specific command is not set in the waypoints menu Options screen.
-gui.xaero_waypoints_edit_mode_individually=One-by-one
-gui.xaero_waypoints_edit_mode_all=All
+gui.xaero_show_angles=Display Facing Angles
+gui.xaero_box_entity_nametags=Display entity name under the dot on the minimap for entities that have name tags.
+gui.xaero_box_teleport_default_command=The command used for waypoint teleportation if a world-specific or a server-specific command is not set in the Options menu on the waypoint screen.
+gui.xaero_waypoints_edit_mode_individually=One by One
+gui.xaero_waypoints_edit_mode_all=All at Once
 gui.xaero_compass=Compass
-gui.xaero_cave_maps_depth=Cave Maps Depth
+gui.xaero_cave_maps_depth=Cave Mode Depth
 gui.xaero_waypoints_edit_reset=Reset
 gui.xaero_main_entity_as=Main Entity As
 gui.xaero_arrow=Arrow
 gui.xaero_dot=Dot
 gui.xaero_crosshair=Crosshair
-gui.xaero_teleport_coordinates_hidden=Unable to teleport because it would tell you the waypoint coordinates. Disable "Hide Waypoint Coords" to be able to freely teleport again.
-gui.xaero_hide_wp_coords=Hide Waypoint Coords
+gui.xaero_teleport_coordinates_hidden=Unable to teleport, because it could expose the waypoint coordinates. Disable the "Hide Waypoint Coordinates" option to be able to freely teleport again.
+gui.xaero_hide_wp_coords=Hide Waypoint Coordinates
 gui.xaero_waypoints_edit_show=Show
 gui.xaero_waypoints_edit_hide=Hide
 gui.xaero_teleport_anyway=Teleport Anyway
-gui.xaero_teleport_shows_coordinates=Shows you the coordinates!
+gui.xaero_teleport_shows_coordinates=It will expose the coordinates!
```

Comment:
Now this is scary haha

Reply:
I wanted to focus on the result of the result of the click, you know, click -> coordinates are shown -> brother/cousin/live viewers see them. I assume I achieved my goal ;)

Reply:
Yeah, this is great :)